src/SprykerSdk/Zed/AppSdk/Business/AppSdkFacadeInterface.php

```diff
@@ -127,4 +129,16 @@ public function buildFromAsyncApi(AsyncApiRequestTransfer $asyncApiRequestTransf
      * @return \Generated\Shared\Transfer\ValidateResponseTransfer
      */
     public function validateAsyncApi(ValidateRequestTransfer $asyncApiRequestTransfer): ValidateResponseTransfer;
+
+    /**
+     * Specification:
```

Comment:
This specification needs to be updated.

Reply:
I updated it.

Reply:
Here you should describe what this method does.

Reply:
I see validateAsyncApi method has specification as 'Reads an AsyncAPI file and validates it.'

And appConfigurationCreate method has specification as 'Adds Configuration file.'.

What detail specification do you want?

Reply:
* - Reads an AsyncAPI file and validates it.
     * - Validate for valid schema
     * - Validate for duplicate message
     * - Validate for missing operationId
     
     Does it make work?

Reply:
I updated detailed specification, could you please check and give your feedback

Reply:
That sounds good to me.